utils/http4s-client/src/test/scala/io/renku/http/rest/SortBySpec.scala

```diff
@@ -89,16 +86,16 @@ class SortBySpec extends AnyWordSpec with ScalaCheckPropertyChecks with should.M
     }
   }
 
-  "by" should {
-
-    "be convertable to QueryParamValue which value is serialized with direction and url encoded" in {
-      val convert = implicitly[TestSort.By => renku.ResourceUrl.QueryParamValue]
-
-      val sort = testSortBys.generateOne
-
-      convert(sort.sortBy.head).value shouldBe urlEncode(serialize(sort.sortBy.head))
-    }
-  }
+//  "by" should {
```

Comment:
Do we have these specs moved somewhere?

Reply:
Thanks! Forgot to remove the tests+code. It is not used (anymore).